### ISSUE TITLE
Mixins don't merge `__classvars__`

### ISSUE THREAD

**Issue by Breush** [wanadev/abitbol#13]
```javascript
var Class = require("abitbol");

var MyClass = Class.$extend({
    
    __include__: [{
        __classvars__: {
            HELLO: "hi",
            SAILOR: "sailor",
        },
    }],

    __classvars__: {
        HELLO: "hello",
    },

});

console.log(MyClass.HELLO);   // "hello"      - Ok
console.log(MyClass.SAILOR);  // undefined    - Not Ok
```

**Comment by flozz** [comment]
implemented in #14

**Comment by flozz** [comment]
Released in v1.2.0
